### PR TITLE
Minor tweaks.

### PR DESCRIPTION
**Pull request by jeromekelleher**
I've made some minor tweaks to split up some long sentences which I thought were confusing.

I've also updated how we talk about Relate and said "inferred trees" rather then "inferred tree sequences" just to try and make the distinction. Comments welcome!

### CODE REVIEW

File: treestats_paper.tex [hunk 1]
```diff
@@ -856,23 +856,24 @@ \section*{Duality of site and branch statistics}
...
+Therefore, if mutations occur with constant rate
+the expected value of genetic diversity
+% averaging over mutational noise given the tree sequence,
+is equal to the mutation rate multiplied by the average path distance between
+the two samples in the trees.
 
 This relationship is true more generally.
 In fact, for any region of the genome between $i$ and $j$,
-with $\treeseq_{[i,j)}$ the trees spanning that interval,
+% with $\treeseq_{[i,j)}$ the trees spanning that interval,
```
Comment (petrelharp) [inline treestats_paper.tex:868]: hm, but we need to define this notation somewhere?

Comment (petrelharp) [reply treestats_paper.tex:868]: maybe we define it in a separate short sentence after the equation?

Comment (petrelharp) [reply treestats_paper.tex:868]: will merge and then deal with this

Comment (jeromekelleher) [reply treestats_paper.tex:868]: I figured we had defined it earlier on, but maybe we didn't. I'm easy either way.

Comment (petrelharp) [reply treestats_paper.tex:868]: I fixed this up in #50.

**Pull request merged**